pkgs/development/interpreters/racket/default.nix

```diff
@@ -47,9 +48,13 @@ stdenv.mkDerivation rec {
 
   FONTCONFIG_FILE = fontsConf;
   LD_LIBRARY_PATH = libPath;
-  NIX_LDFLAGS = stdenv.lib.optionalString stdenv.cc.isGNU "-lgcc_s";
+  NIX_LDFLAGS = stdenv.lib.concatStringsSep " " [
+    (stdenv.lib.optionalString (stdenv.cc.isGNU && ! stdenv.isDarwin) "-lgcc_s")
```

Comment:
Isn't just `isGNU` enough?

Reply:
It is possible, if you want to for whatever reason, to build this package with gcc7Stdenv, just pass that as the stdenv, and in case you do, you need to not use -lgcc_s.

Reply:
(on darwin)

Reply:
That's not something we usually worry about, but I guess it doesn't hurt.